src/modules/block/components/blocks/blocks.css

```diff
@@ -42,3 +42,29 @@
     text-align: right;
   }
 }
+
+.header {
+  width: 100%;
+  height: calc(var(--header-height) + 45px);
+
+  & > h1 {
+    margin-bottom: 12px;
+  }
+
+  & ~ div:nth-child(2) {
```

Comment:
This is also pretty tricky. Is this the only way to address your target element?